examples/testhandler/tests/test_handler.py

```diff
@@ -137,3 +140,10 @@ def test_close_cache(project):
     versions = handler.cache.counterforVersion.keys()
     assert len(versions) == 0
     assert len(handler.cache.cache) == 0
+
+
+def test_sys_executable(project):
```

Comment:
```suggestion
def test_280_sys_executable(project):
```
The motivation for this test is not trivial, I think linking to the ticket is useful.